Thilaga-26/vyos-1x
# T6358: Unused imports using pylint

Title: T6358: Unused imports using pylint

## Summary
This PR corrects the unused imports file and updates the reusable actions

## Details of Changes
- The unused imports file has been changed as reusable action.

## Testing Steps
1. Clone the repository.
2. Navigate to the `unused imports.yml` file.
3. Verify that the updated reusable actions file.

## Additional Notes
Please ensure to review the updated reusable actions file carefully as they include reusable steps in the project.

## Checklist
